relay-server/src/endpoints/forward.rs

```diff
@@ -112,8 +146,8 @@ impl fmt::Debug for ForwardRequest {
 }
 
 impl UpstreamRequest for ForwardRequest {
-    fn method(&self) -> Method {
-        self.method.clone()
+    fn method(&self) -> reqwest::Method {
+        self.method.as_str().parse().unwrap()
```

Comment:
should we fallback to some default method instead of `unwrap` here ?

Reply:
Good question. The panic here is unreachable, since the parsed method is already valid. A default would also create an invalid request.